src/ripple/basics/TaggedCache.h

```diff
@@ -186,8 +186,7 @@ class TaggedCache
             }
             else
             {
-                when_expire = now - clock_type::duration (
-                    m_target_age.count() * m_target_size / m_cache.size ());
+                when_expire = now - m_target_age*m_target_size/m_cache.size();
```

Comment:
Nit: spaces around operators

Reply:
The spaces will push the line past column 80, and I figured it was more readable without the spaces than to break the line.